NineChronicles.Headless/GraphTypes/ActionQueryFields/CombinationConsumable.cs

```diff
@@ -0,0 +1,48 @@
+using GraphQL;
+using GraphQL.Types;
+using Libplanet;
+using Libplanet.Explorer.GraphTypes;
+using Nekoyume.Action;
+using NCAction = Libplanet.Action.PolymorphicAction<Nekoyume.Action.ActionBase>;
+
+namespace NineChronicles.Headless.GraphTypes;
+
+public partial class ActionQuery
+{
+    private void RegisterCombinationConsumable()
+    {
+        Field<NonNullGraphType<ByteStringType>>(
+            "CombinationConsumable",
+            arguments: new QueryArguments(
+                new QueryArgument<NonNullGraphType<AddressType>>
+                {
+                    Name = "avatarAddress",
+                    Description = "Avatar address to combine consumable"
+                },
+                new QueryArgument<NonNullGraphType<IntGraphType>>
+                {
+                    Name = "slotIndex",
+                    Description = "Slot index to combine"
+                },
+                new QueryArgument<NonNullGraphType<IntGraphType>>
+                {
+                    Name = "recipeId",
+                    Description = "Recipe ID to combine consumable"
+                }
+            ),
+            resolve: context =>
+            {
+                var avatarAddress = context.GetArgument<Address>("avatarAddress");
+                var slotIndex = context.GetArgument<int>("slotIndex");
+                var recipeId = context.GetArgument<int>("recipeId");
+                NCAction action = new CombinationConsumable
+                {
+                    avatarAddress = avatarAddress,
+                    slotIndex = slotIndex,
+                    recipeId = recipeId
```

Comment:
```suggestion
                    recipeId = recipeId,
```

additionally, it also would be nice with linter to check [SA1413](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/2f6d02f95d34a771717d9d4dab5d1b8819e3e5dd/documentation/SA1413.md).